### ISSUE TITLE
mzcompose sometimes fails if not run from the root

### ISSUE THREAD

**Issue by benesch** [MaterializeInc/materialize#3300]
This mzconduct command triggers the failure for @umanwizard:

```
~/c/materialize ❮❮❮ bin/mzconduct run chaos -w stop-kafka                                                                                                                       ✘ 1 
C> Executing chaos -> stop-kafka
$ ./mzcompose --mz-quiet -f ./mzcompose.yml up -d materialized kafka schema-registry dashboard
$ docker pull materialize/materialized:mzbuild-SLUTI4XSMF2IUWM37CYQU2S67RTRWLC4
Error response from daemon: manifest for materialize/materialized:mzbuild-SLUTI4XSMF2IUWM37CYQU2S67RTRWLC4 not found: manifest unknown: manifest unknown
$ git clean -ffdX /Users/brennan/code/materialize/src/materialized/ci
Removing ../../src/materialized/ci/materialized
$ /Users/brennan/code/materialize/bin/xcompile build --release --bin materialized
    Finished release [optimized + debuginfo] target(s) in 0.62s
$ x86_64-unknown-linux-gnu-objcopy -R .debug_pubnames -R .debug_pubtypes /Users/brennan/code/materialize/src/materialized/ci/materialized
error: no bin target named `materialized`
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/brennan/code/materialize/misc/python/materialize/cli/mzcompose.py", line 183, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/Users/brennan/code/materialize/misc/python/materialize/cli/mzcompose.py", line 53, in main
    composes.append(build_compose_file(repo, args.command, config_file))
  File "/Users/brennan/code/materialize/misc/python/materialize/cli/mzcompose.py", line 118, in build_compose_file
    deps.acquire()
  File "/Users/brennan/code/materialize/misc/python/materialize/mzbuild.py", line 602, in acquire
    acquired_from = d.acquire()
  File "/Users/brennan/code/materialize/misc/python/materialize/mzbuild.py", line 459, in acquire
    self.build()
  File "/Users/brennan/code/materialize/misc/python/materialize/mzbuild.py", line 433, in build
    self.image.pre_image.run()
  File "/Users/brennan/code/materialize/misc/python/materialize/mzbuild.py", line 229, in run
    self.build()
  File "/Users/brennan/code/materialize/misc/python/materialize/mzbuild.py", line 215, in build
    cargo_build + ["--message-format=json"], unicode=True
  File "/Users/brennan/code/materialize/misc/python/materialize/spawn.py", line 125, in capture
    args, cwd=cwd, input=stdin, universal_newlines=unicode, stderr=stderr
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 512, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/Users/brennan/code/materialize/bin/xcompile', 'build', '--release', '--bin', 'materialized', '--message-format=json']' returned non-zero exit status 101.
C> Failed! ERROR: services didn't come up cleanly: materialized, kafka, schema-registry, dashboard
```

**Comment by wangandi** [comment]
I'm also encountering this problem.

**Comment by benesch** [comment]
I believe I fixed this with f5770b9.
